src/herumi/signature.ts

```diff
@@ -17,7 +17,8 @@ export class Signature implements ISignature {
     this.value = value;
   }
 
-  static fromBytes(bytes: Uint8Array): Signature {
+  static fromBytes(bytes: Uint8Array, _type?: CoordType, validate = true): Signature {
+    if (validate === false) throw new Error("ValidationSkipNotSupported");
```

Comment:
After thinking about this check from an implementer point of view I'm a bit worried about it.

Lodestar may implement a feature where re-processed blocks are not signature validated so this becomes false. However that situation may only happen during deep re-orgs and when a user can't compile BLST. So randomly the node will stall in an opaque way.

I would recommend to remove and add notes in the JSDoc comments:
```
When using `herumi` implementation, signature validation is always on regardless of this flag.
```